apis/python/src/tiledbsoma/_sparse_nd_array.py

```diff
@@ -266,27 +266,45 @@ def read(
             * Negative indexing is unsupported.
         """
         del batch_size  # Currently unused.
-        handle: clib.SOMASparseNDArray = self._handle._handle
-
         self._check_open_read()
         _util.check_unpartitioned(partitions)
 
-        context = handle.context()
-        if platform_config is not None:
-            config = context.tiledb_config.copy()
-            config.update(platform_config)
-            context = clib.SOMAContext(config)
-
-        sr = clib.SOMASparseNDArray.open(
-            uri=handle.uri,
-            mode=clib.OpenMode.read,
-            context=context,
-            column_names=[],
+        return SparseNDArrayRead(
+            array=self,
+            coords=coords,
             result_order=_util.to_clib_result_order(result_order),
-            timestamp=handle.timestamp and (0, handle.timestamp),
+            platform_config=platform_config,
         )
 
-        return SparseNDArrayRead(sr, self, coords)
+    def resize(
```

Comment:
Weird that these are showing up new

... I know what happened though:

* We had support for current domain for sparse arrays in core 2.26
* We had support for current domain for dense arrays in core 2.27
* So I had these two methods in `apis/python/src/tiledbsoma/_sparse_nd_array.py` and `apis/python/src/tiledbsoma/_dense_nd_array.py`
* Now that we're on 2.27.0rc3 these don't need to be duplicated anymore
* On #3399 I pulled these down to `apis/python/src/tiledbsoma/_common_nd_array.py` (probably that was confusing of me to have done so on that PR, given that PR's title)

Reply:
Anyway please remove these, and/or double-check your rebase -- #3399 is how things should be moving forward